### ISSUE TITLE
krakend fails to compile with go 1.11

### ISSUE THREAD

**Issue by fxcoudert** [devopsfaith/krakend-ce#37]
krakend fails to compile with go 1.11:

```
==> make build
Building the binary...
# github.com/devopsfaith/krakend-ce
github.com/devopsfaith/krakend-ce/vendor/github.com/tmthrgd/go-hex.encodeASM: relocation target runtime.support_avx not defined
github.com/devopsfaith/krakend-ce/vendor/github.com/tmthrgd/go-memset.memsetAsm: relocation target runtime.support_avx not defined
make: *** [build] Error 2
```

https://github.com/Homebrew/homebrew-core/pull/31887

**Comment by taik0** [comment]
Hi @fxcoudert 

In go 1.11 the removed some functions from the runtime package and some of the deps versions still use those runtime functions.

Use [this branch](https://github.com/devopsfaith/krakend-ce/tree/go111_mac_fix) to fix this particular issue.

**Comment by fxcoudert** [comment]
Homebrew ships released versions of software. We'll need a new release with the fix in, to continue distributing krakend

**Comment by taik0** [comment]
Oh, I've missed the homebrew part. I'm looking into it.

**Comment by alombarte** [comment]
Thanks, @fxcoudert 

Fixed in 0.6.1, live https://formulae.brew.sh/formula/krakend !
